src/SDK/SdkBuilder.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK;
+
+use OpenTelemetry\API\Metrics\MeterProviderInterface;
+use OpenTelemetry\API\Metrics\Noop\NoopMeterProvider;
+use OpenTelemetry\API\Trace\NoopTracerProvider;
+use OpenTelemetry\API\Trace\TracerProviderInterface;
+use OpenTelemetry\Context\Propagation\TextMapPropagatorInterface;
+
+class SdkBuilder
+{
+    private ?TracerProviderInterface $tracerProvider = null;
+    private ?MeterProviderInterface  $meterProvider = null;
+    private array $propagators = [];
+
+    public function setTracerProvider(TracerProviderInterface $provider): self
+    {
+        $this->tracerProvider = $provider;
+
+        return $this;
+    }
+
+    public function setMeterProvider(MeterProviderInterface $meterProvider): self
+    {
+        $this->meterProvider = $meterProvider;
+
+        return $this;
+    }
+
+    public function setPropagator(TextMapPropagatorInterface $propagator): self
+    {
+        $this->propagators[] = $propagator;
+
+        return $this;
+    }
+
```

Comment:
java's implementation also has a `buildAndRegisterGlobal()` method. A similar method here could forward on to `OpenTelemetry\API\Common\Instrumentation\Configurator` ?